update_version.sh

```diff
@@ -52,9 +52,12 @@ sed -i "s/^\(securedrop_app_code_version: \"\).*/\1$NEW_VERSION\"/" install_file
 # Update the version in molecule testinfra vars
 sed -i "s@$(echo "${OLD_VERSION}" | sed 's/\./\\./g')@$NEW_VERSION@g" molecule/builder/tests/vars.yml
 
-# Update the version that we tell people to check out in the install doc
-sed -i "s@$(echo "${OLD_VERSION}" | sed 's/\./\\./g')@$NEW_VERSION@g" docs/set_up_admin_tails.rst
-sed -i "s@$(echo "${OLD_VERSION}" | sed 's/\./\\./g')@$NEW_VERSION@g" docs/conf.py
+# Update the version that we tell people to check out in the install doc if an actual release.
+# We should not update version string in the docs with a release candidate is created
+if [[ ! $NEW_VERSION == *~rc* ]]; then
+  sed -i "s@$(echo "${OLD_VERSION}" | sed 's/\./\\./g')@$NEW_VERSION@g" docs/set_up_admin_tails.rst
+  sed -i "s@$(echo "${OLD_VERSION}" | sed 's/\./\\./g')@$NEW_VERSION@g" docs/conf.py
```

Comment:
So there's the following scenario:

1. Bump 0.11.0 to 0.11.0-rc1: docs don't get updated as expected, so docs say 0.11.0
2. Bump 0.11.0-rc1 to 0.12.0: docs don't get updated since this script is using the 0.11.0-rc1 version string from `__version__.py`, i.e. not 0.11.0

We could do something like just taking the beginning part of the version string before `~rc` from the OLD_VERSION and _then_ apply these sed commands - what do you think?

Reply:
Ah right, good point. What about using `molecule/shared/stable.ver`? I've updated the PR and the test plan based on your comments and it should address your comments